packages/core/src/middleware/koa-console-redirect-proxy.test.ts~

```diff
@@ -0,0 +1,43 @@
+import { MockQueries } from '#src/test-utils/tenant.js';
```

Comment:
can write your test in a single test file

Reply:
i don’t understand, the test is written in only one test file, i am only importing a method used for mocking queries.